frontend/public/components/overview/cron-job-overview.tsx

```diff
@@ -17,30 +18,41 @@ import { JobsOverview } from './jobs-overview';
 
 const CronJobOverviewDetails: React.SFC<CronJobOverviewDetailsProps> = ({
   item: { obj: cronjob },
-}) => (
-  <div className="overview__sidebar-pane-body resource-overview__body">
-    <div className="resource-overview__pod-counts">
-      <PodRingSet key={cronjob.metadata.uid} obj={cronjob} path="" />
-    </div>
-    <ResourceSummary resource={cronjob} showPodSelector>
-      <DetailsItem label="Schedule" obj={cronjob} path="spec.schedule" />
-      <DetailsItem label="Concurrency Policy" obj={cronjob} path="spec.concurrencyPolicy" />
-      <DetailsItem
-        label="Starting Deadline Seconds"
-        obj={cronjob}
-        path="spec.startingDeadlineSeconds"
-      >
-        {cronjob.spec.startingDeadlineSeconds
-          ? pluralize(cronjob.spec.startingDeadlineSeconds, 'second')
-          : 'Not Configured'}
-      </DetailsItem>
-      <DetailsItem label="Last Schedule Time" obj={cronjob} path="status.lastScheduleTime">
-        <Timestamp timestamp={cronjob.status.lastScheduleTime} />
-      </DetailsItem>
-    </ResourceSummary>
-  </div>
-);
+}) => {
+  const { t } = useTranslation();
 
+  return (
+    <div className="overview__sidebar-pane-body resource-overview__body">
+      <div className="resource-overview__pod-counts">
+        <PodRingSet key={cronjob.metadata.uid} obj={cronjob} path="" />
+      </div>
+      <ResourceSummary resource={cronjob} showPodSelector>
+        <DetailsItem label={t('public~Schedule')} obj={cronjob} path="spec.schedule" />
+        <DetailsItem
+          label={t('public~Concurrency policy')}
+          obj={cronjob}
+          path="spec.concurrencyPolicy"
+        />
+        <DetailsItem
+          label={t('public~Starting deadline seconds')}
+          obj={cronjob}
+          path="spec.startingDeadlineSeconds"
+        >
+          {cronjob.spec.startingDeadlineSeconds
+            ? pluralize(cronjob.spec.startingDeadlineSeconds, 'second')
```

Comment:
We don't want to use pluralize here since it won't be internationalized. Instead you can use react-i18next to pluralize it like this:
`t('public~{startingDeadlineSeconds} second', {cronjob.spec, count})`

Reply:
`t('public~{{count}} second', {count: cronjob.spec.startingDeadlineSeconds})`